src/pages/index.tsx

```diff
@@ -35,12 +35,12 @@ const IndexPage: React.FC = () => (
     <PageLinks />
     <GeneralBodyContainer>
       <GeneralDescription
-        title="Board Applications Open!"
+        title="Welcome to VandyHacks!"
         text1={
-          `Welcome to VandyHacks! Our 2021/VH VIII Board applications are currently open until 11:59 PM CST on January 20th, 2021.`
+          `Our 2021/VH VIII Board applications have closed, and we are ecstatic about all our new members!`
         }
         text2={
-          `Join us in making Vanderbilt University's computer science community an even more enjoyable, diverse, and welcoming space.`
+          `Click here to learn more about VandyHacks and what we do as an organization. We hope to see your application for VH XI!`
```

Comment:
bro XI is 11, we want VIII for 8 ig unless it's for like internal recruitment for VH IX

Reply:
yeah ig we need IX

Reply:
LOL wait I got IX and XI mixed